### ISSUE TITLE
Got it working in 2.91

### ISSUE THREAD

**Issue by schroef** [meshlogic/extra-material-list#1]
I bumped into your site again after working a bit more on Extra Image List. As i said on your website, i wanted to be able to send it back to nodes. I first wanted the panel to show in the node editor, but i didnt know what data i needed. So after i checked your Extra Material List i got an idea.

I know have your Materiallist addon working and i added the Image list as well. So when your working on a material you can also easily pick a texture. Perhaps could add the rest of the UI as well. But if we/you want to keep them separate thats okay of course. 

I think it needs some thought of UX and UI design if they are merged to one in the Node editor.

It does work and is super nice to work with :)

![extra-mat-list-withimage](https://user-images.githubusercontent.com/6923008/113836774-30b63e00-975b-11eb-8b25-9cdfec9fdf98.png)

**Comment by Razboy20** [comment]
@schroef just wondering, do you have the updated script on hand/on a forked GitHub repository?

**Comment by schroef** [comment]
Yeah i still have the code. Was side tracked with different add-ons. I'll try to make a repo and post s link. Not sure I'll do a pull request because my implementation nis so much different

**Comment by schroef** [comment]
> @schroef just wondering, do you have the updated script on hand/on a forked GitHub repository?


I had already made a branch, kinda forgot about it. Im not sure how it holds up in 3.0. I have not yet tried that version yet
https://github.com/schroef/extra-material-list/tree/Adjustments-Panel-Layout-bl291

I did add some more parts like material slots and other items. Together with Amarant, double click node to show image texture in UV editor, its super nice and smooth experience
